src/Toggle/UseCase/ToggleRetriever.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Clearbooks\Labs\Toggle\UseCase;
+
+use Clearbooks\Labs\Db\Entity\Toggle;
+
+interface ToggleRetriever
+{
+    /**
+     * @param int $toggleId
+     * @return Toggle|null
```

Comment:
Hm I'd rather not write getter and setter interfaces for every data holding object, that would go wild really quickly, wouldn't you think? :S I feel like this level of separation is ok for a service but it feels a little bit too much for an object like the Toggle.